README.md

```diff
@@ -101,12 +116,31 @@ git commit --amend                      # Re-do the commit message of the previo
 - or for instance Notepad++:
 `git config --global core.editor "'C:/Program Files/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`
 
+# Re/moving files under version control
+git rm <path/to/the/file>                 # remove file and stage the change to be committed.
+git mv <source/file> <destination/file>   # move/rename file and stage the change to be committed.  
 
 
 # See history
 git log             # Show commit logs
 git log --oneline   # Formats commits to a single line (shorthand for --pretty=oneline  --abbrev-commit )
 git log --graph     # Show a graph commits and branches
+git log --pretty=fuller     # To see commit log details with author and committer details, if any different.
+git log --follow <file>     # List history of given file.
+git log branch2..branch1    # Show commits present on branch1 compared to branch1
+
+# Deferring
```

Comment:
If stash should even be included here, it should be further down.

I am a bit in doubt on how much we want to include here in the README. I think it should just be the basics, but then we could have a more extensive CHEATSHEET.md with a more thorough list?

Reply:
Taken away.